includes/admin/functions.php

```diff
@@ -166,3 +166,13 @@ function amp_editor_core_blocks() {
 	$editor_blocks = new AMP_Editor_Blocks();
 	$editor_blocks->init();
 }
+
+/**
+ * Bootstrap the AMP admin pointer class.
+ *
+ * @since 1.0
+ */
+function amp_admin_pointer() {
+	$admin_pointer = new AMP_Admin_Pointer();
+	$admin_pointer->init();
+}
```

Comment:
This bootstrapping is repetitive. It'd be nice if `amp_post_meta_box()`, `amp_editor_core_blocks()`, and `amp_admin_pointer()` could be combined into one function, like `amp_admin_bootstrap()`. 

But in the unlikely case that a user [unhooked](https://github.com/Automattic/amp-wp/pull/1271/files#diff-4805c837d3390517773926fb26726bcdL155) one of these from `wp_loaded`, this would break that.

Reply:
Then the questions are:  

- Do we need to provide the ability to unhook any of those components? 
- Do we need to provide the ability to unhook any of these components?

```
	add_action( 'wp_loaded', 'amp_editor_core_blocks' );
	add_action( 'wp_loaded', 'amp_post_meta_box' );
	add_action( 'wp_loaded', 'amp_editor_core_blocks' );
	add_action( 'wp_loaded', 'amp_add_options_menu' );
	add_action( 'wp_loaded', 'amp_admin_pointer' );
```

Changing that to `add_action( 'wp_loaded', 'amp_admin_bootstrap' );` and then letting `amp_admin_bootstrap()` manage the order and initializations results in less callbacks having to be processed, i.e. only 1 instead of 5.  It'll be slightly faster.

Reply:
Good questions. I think it'd be best to avoid renaming the functions above, as a user could have unhooked them. 

But it's not a strongly-held belief, and there probably aren't many users who would have unhooked them.

Another option is renaming `amp_admin_pointer()` to `amp_admin_bootstrap()`, without making any other change.

Then, any future admin classes could add their bootstrapping there.

Reply:
>Another option is renaming amp_admin_pointer() to amp_admin_bootstrap(), without making any other change.

This option is future-thinking as you are planning for future bootstrapping functionality.  That's good.

Now, one could argue that confusion may be introduced when a dev is deciding where to bootstrap.  Adding an inline comment above the action hook may elevate that confusion.

Reply:
I'd vote to review how we want to bootstrap and consider what callbacks we can consolidate.  That said, I think it's outside the scope of this particular PR.  We may want to open a ticket for it and begin the discussion process beyond ours here.

Reply:
Sounds good, Tonya.

Reply:
Yes, I think we should try to get away from bootstrapping like this because the class instantiation variable that gets created is not accessible anywhere. We should instead of an AMP plugin manager that is responsible for instantiating classes and then keeping around the references to the class instances for the plugin (and others) to access later.